src/main/java/module-info.java

```diff
@@ -82,4 +82,11 @@
     requires org.antlr.antlr4.runtime;
     requires flowless;
     requires org.apache.tika.core;
+
+    requires flexmark;
+    requires flexmark.ext.gfm.strikethrough;
+    requires flexmark.ext.gfm.tasklist;
+    requires flexmark.util.ast;
+    requires flexmark.util.data;
+    requires annotations;
```

Comment:
is `requires annotations` really necessary. I remember that I had a few problems with annotations vs `jakarta.annotation` in the past.

Reply:
You're right it's not needed, I was tempted to put `@NotNull` annotations on parameters but removed them because it doesn't make sense to use them on one class only. Apparently forgot to remove the `requires.